scripts/workflows/release/run-release-with-gates.cjs

```
@@ -0,0 +1,125 @@
+  callPhase4();
+  process.exit(0);
+}
+
+function log(message, level = 'INFO') {
+  const timestamp = new Date().toISOString();
+  console.log(`[${timestamp}] ${level}: ${message}`);
+}
+
+function callPhase4() {
+  log('Calling Phase 4 release agent (run-release-agent.cjs)');
+  const phase4Script = path.join(__dirname, 'run-release-agent.cjs');
```

ESLint

[error] '__dirname' is not defined.

(no-undef)